net/fail2ban/Makefile

```diff
@@ -0,0 +1,112 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fail2ban
+PKG_VERSION:=0.11.2
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/fail2ban/fail2ban/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=383108e5f8644cefb288537950923b7520f642e7e114efb843f6e7ea9268b1e0
+
+PKG_BUILD_DEPENDS:=python3/host
+HOST_BUILD_DEPENDS:=python3-lib2to3/host
+
+include $(INCLUDE_DIR)/package.mk
+$(call include_mk, python3-package.mk)
```

Comment:
This is not correct.

Reply:
fixed by moving my packages inside a local fork ... thanks